requirements.txt

```diff
@@ -1,18 +1,8 @@
-backports.lzma==0.0.8
-click==6.7
-cryptography==2.3
+click>=6.7
 cloud-volume>=0.38.0
 google-cloud-logging
-google-api-python-client==1.6.2
-networkx==2.1
 numpy>=1.14.2
-pyasn1
-pyopenssl>=17.5.0
 pytest>=3.3.1
-scipy>=1.1.0
 task-queue>=0.4.1
-tenacity>=4.10.0
 -e git+https://github.com/seung-lab/tqdm.git#egg=tqdm
-oauth2client==3.0.0
-protobuf>=3.5.1
-python-jsonschema-objects==0.3.3
```

Comment:
The tasks depends on these packages?
Will igneous still work without installing them?

Reply:
I looked through the code for any direct imports of those packages within igneous, but didn't see any. Most of them are a dependency of `cloud-volume` or `task-queue` (e.g. `tenacity`, `networkx`, `pythoin-jsonschema-objects`) and as such should be handled in these two projects. And a few others are simply not used anymore (e.g. `backports.lzma` was part of the old, out-commented `HyperSquareTask`)